homework-assignments-1/task-3.js

```diff
@@ -0,0 +1,45 @@
+class Book {
+    constructor(title, author, isbn){
+        this.title = title;
+        this.author = author;
+        this.isbn = isbn;
+    }
+}
+
+class Library{
+    constructor(books){
```

Comment:
When you create instance of Library, it will be empty so you don't need to add constructor for it.
private books: Book[] = []; // this will do the work.